docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -14,7 +14,7 @@ Requirements
 
 To connect to MongoDB, you need:
 
-* MongoDB 4.0 or higher.
+* MongoDB 4.2 or higher.
```

Comment:
This should go into release notes @colebow

Reply:
Is there something I need to do for that?

Reply:
Nope, you're good! You've ticked the box in the PR template, so I've got it from here. 😄